website/client/src/components/achievements/streak.vue

```diff
@@ -29,12 +29,14 @@
           {{ $t('dontStop') }}
         </button>
         <div class="checkbox">
-          <input
-            v-model="user.preferences.suppressModals.streak"
-            type="checkbox"
-            @change="suppressModals"
-          >
-          <label>{{ $t('dontShowAgain') }}</label>
+          <label>
```

Comment:
thanks @cstyles , I think that for accessibility reasons it would be better to keep the label and input as siblings while adding an unique `id` attribute to the input and a matching `for` attribute to the label meaning that the label is linked to the input when using a screen readeer, etc...

Putting the input inside the labels works as well but the w3 site suggests using explicit labels (https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly) and Bootstrap's styles work better with the label used as a sibling of the input (https://getbootstrap.com/docs/4.3/components/forms/)

Reply:
Yeah, I would have preferred that too but as I'm not familiar with the codebase I didn't know what to pick to avoid conflicts. Would something as simple as `dont-show-again` suffice?

Reply:
I would probably use something that matches the user preference so in this case `user-preference-suppressModals-streak`

Reply:
OK, I just pushed the change. Let me know what you think!